### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Relequestual**
Direct to org wide support document

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Issues
 
-Issues should identify an problem, enhancement, or use case; and propose some course of action. For alternate support channels, [see the json-schema.org website](http://json-schema.org/) or the [jsonschema tag on stackoverflow](https://stackoverflow.com/tags/jsonschema).
+Issues should identify an problem, enhancement, or use case; and propose some course of action. For alternate support channels, our [support document](https://github.com/json-schema-org/.github/blob/main/SUPPORT.md).
```
Comment (Julian) [inline CONTRIBUTING.md]:
```suggestion
Issues should identify an problem, enhancement, or use case; and propose some course of action. For alternate support channels, see our [support document](https://github.com/json-schema-org/.github/blob/main/SUPPORT.md).
```

Comment (Relequestual) [reply CONTRIBUTING.md]: *sigh* thank YOU!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Issues
 
-Issues should identify an problem, enhancement, or use case; and propose some course of action. For alternate support channels, [see the json-schema.org website](http://json-schema.org/) or the [jsonschema tag on stackoverflow](https://stackoverflow.com/tags/jsonschema).
+Issues should identify an problem, enhancement, or use case; and propose some course of action. For alternate support channels, see our [support document](https://github.com/json-schema-org/.github/blob/main/SUPPORT.md).
```
Comment (Julian) [inline CONTRIBUTING.md]: I guess while we're here also
```suggestion
Issues should identify a problem, enhancement, or use case; and propose some course of action. For alternate support channels, see our [support document](https://github.com/json-schema-org/.github/blob/main/SUPPORT.md).
```

**Review by Julian — APPROVED** [review]

**Review by gregsdennis — APPROVED** [review]
I concur with Julian that the typo should be fixed.

**Pull request merged**